.github/workflows/deploy-dev.yaml

```diff
@@ -1,109 +1,135 @@
-name: Deploy Dev
+name: Deploy DUP-Admin Dev
+
+run-name: Deploying ${{ github.ref_name }} (Admin) to dev
 
 on:
   push:
     branches: [main]
-
-env:
-  TF_VERSION: 0.14.7
-  TG_VERSION: 0.37.1
-  TG_SRC_PATH: terraform
-  TFC_WORKSPACE: dev
-  TARGET_ENV: dev
+    paths: "dup-admin/**"
```

Comment:
I'm not sure this will work, there is no `dup-admin/*` path in this repo?

Reply:
You are right there is not..... I should have caught that one. Originally all the SAM files were in a dup-admin/ dir and when I pulled out terraform moved all the SAM stuff up one level. 

Ill make the updates.